static/js/buddy_data.js

```diff
@@ -250,13 +250,15 @@ export function get_title_data(user_ids_string, is_group) {
     // a single, human, user.
     const active_status = presence.get_status(user_id);
     const last_seen = user_last_seen_time_status(user_id);
+    const is_me = people.is_my_user_id(user_id);
 
     // Users has a status.
     if (user_status.get_status_text(user_id)) {
         return {
             first_line: person.full_name,
             second_line: user_status.get_status_text(user_id),
             third_line: get_last_seen(active_status, last_seen),
+            show_you: is_me,
```

Comment:
I think `is_my_user` would be a better name for this variable, for similarity with the `people.js` function.

(As a general rule, a line like this that renames a variable is probably not what you want -- one or the other probably has the wrong name)